dataactbroker/handlers/fileHandler.py

```diff
@@ -1176,7 +1176,7 @@ def get_error_metrics(submission):
         return JsonResponse.error(e, StatusCode.INTERNAL_ERROR)
 
 
-def list_submissions(page, limit, certified):
+def list_submissions(page, limit, certified, sort='modified', order='desc'):
```

Comment:
good call, fileRoutes.py has default values set incase of missing args so it should be fine to remove